DESCRIPTION

```diff
@@ -51,7 +52,6 @@ Suggests:
     data.table,
     epidatr (>= 1.0.0),
     fs,
```

Comment:
Did `fs` get accidentally added back in after the "prune imports" merge?

Reply:
No this is correct. It should be in Suggests (got pruned from Imports).